### DISCUSSION TITLE
Anime Dub Replace/Additional Language Support

### DISCUSSION THREAD

**Discussion by XxUnkn0wnxX** [pymedusa/Medusa#6238] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
its very frustrating to manually search for dub replacements episodes of anime every time they come out. especially when you have a library of 190+ shows.... 30+ anime.

EG like Date a live, Funimation dubs the previous EP every 2 weeks after the jap ver air's.. kinda frustrating to keep track when you have a lot of anime that is dubbed... or going to be dubbed...

**Describe the solution you'd like**
Add Support for Anime Dubs to be downloaded once released to replace jap ver.

**Additional context**
so like how propers work for shows, when a dubbed version of an episode or season is out it. medusa should replace the jap ver with the dub.

Dub Language Preferences: English, Italian, Dutch etc.. (in edit page of a show)

this could also be added to the edit function of all shows not just anime incase some of us have libraries in different languages & prefer dubs in other languages.

also add Show Language where "Language Info" is if required.
probably also add another column to the show/anime. Label: Dubbed Language
Possibilities: English, Italian, Dutch etc...

could probably leverage preferred words in someway but also make preferred words a parameter that is editable for each show not just as a global option..

**Comment by XxUnkn0wnxX** [comment]
let me expand/revise the original idea.
have an additional searcher like Daily, Proper Backlog etc

but configurable per show only
if enabled it will respect the Allowed/Preferred Qualities, Scene exceptions etc but it will always replace a allowed/preferred quality for any words it finds.

say you include `dub, dual, eng` in the wordlist, if it finds any of these with the show name/scene name. it will replace it.
while respecting the allowed/preferred qualities. so like normal search & proper but will keep searching till it finds an episode with any of the included words then stop for that ep when replaced.

obviously wont run if a show is paused or for an EP that is archived. 

EG:

Black Clover Anime
Daily, proper will do their thing and DL the show as is, Respecting Allowed/Preferred qualities.

then Filtered search runs after 2-3 days from EP air time using the word list: `dub, dual, eng`
Black Clover S01E151 web-dl downloaded, will continually search till a word from word list is matched then
replace Black Clover S01E151 web-dl with Black Clover S01E151 `dub` hdtv.
search finished for Black Clover S01E151.

so essentially its a customisable Proper searcher which will replace EP no matter what once it finds a word from the word list then stop.

**Reply by p0psicles** [reply]
This is basically the same feature request as the 'upgrade results based on prefered words' right?

**Reply by XxUnkn0wnxX** [reply]
ya, I just kept revising it over the months/years trying to simplify it as possible..
but I didn't want a global search, has be be configurable per show bases similar like how you have release groups for anime per show bases. but have configurable searcher with key words ontop of the usual daily searcher per show.

& this searcher/preferred words will supersede anything already downloaded till it matches the word from the word list..

like replacing Allowed quality with preferred, but it this case replacing Daily's & propers with keyword matched instead..

other scenarios could include say, download English show, but then later replace English show with multiple languages etc.. the keyword would be `Multi-Audio`

while global preferred is ok, once a show has been downloaded with preferred quality it wont search anymore like when dubs or other languages come out..
